src/dashboard/apps/consent/management/commands/duplicateconsents.py

```diff
@@ -0,0 +1,33 @@
+"""Consent management duplicate consents commands."""
+
+import sentry_sdk
+
+from apps.consent.helpers import duplicate_expiring_consents
+from apps.core.management.commands.base_command import DashboardBaseCommand
+
+
+class Command(DashboardBaseCommand):
+    """Command for duplicating expiring consents."""
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        """Executes the command for duplicating expiring consents."""
+        self.duplicate_expiring_consents()
+
+    def duplicate_expiring_consents(self):
+        """Duplicate expiring consents that are nearing expiration.
+
+        This method attempts to duplicate expiring
+        consents and logs an error message if the operation fails.
+
+        Raises:
+            Exception: Captures any exception occurring during the consent
+            duplication process and logs the error, including the relevant SIRET and
+            error details.
+        """
+        try:
+            duplicate_expiring_consents()
```

Comment:
It would be beneficial to have the ability to trigger the order for one, multiple, or all entities, using a SIRET number as a parameter.